pkg/qemu/qemu.go

```diff
@@ -812,6 +835,75 @@ func Cmdline(cfg Config) (string, []string, error) {
 	return exe, args, nil
 }
 
+func FindVirtiofsd() (string, error) {
+	type vhostUserBackend struct {
+		BackendType string `json:"type"`
+		Binary      string `json:"binary"`
+	}
+
+	const vhostConfigsDir = "/usr/share/qemu/vhost-user"
+
+	configEntries, err := os.ReadDir(vhostConfigsDir)
+	if err != nil {
+		return "", err
+	}
+
+	for _, configEntry := range configEntries {
+		logrus.Debugf("checking vhost config %s", configEntry.Name())
+		if !strings.HasSuffix(configEntry.Name(), ".json") || !configEntry.Type().IsRegular() {
```

Comment:
No support for symlinks?

Reply:
ngl I'm not sure why I put that, removed it now